travis/install_pg

```diff
@@ -4,13 +4,31 @@
 
 set -eux
 
-# always install postgresql-common
-packages="postgresql-common libedit-dev libpam0g-dev libselinux1-dev"
+# If USE_CUSTOM_PG_BUILD variable is not set, we will assume custom build is
+# not required. If custom build is not required, we will install PostgreSQL
+# via apt, otherwise we will compile from the source.
+if [ -z "$USE_CUSTOM_PG_BUILD" ] || ! [ $USE_CUSTOM_PG_BUILD ]; then
```

Comment:
I used `[ "${USE_CUSTOM_PG_BUILD-0}" = "0" ]`. Parameter substitution also handles if `USE_CUSTOM_PG_BUILD` is not set.